libs/ledger-live-common/src/families/bitcoin/deviceTransactionConfig.ts

```diff
@@ -26,6 +26,13 @@ function getDeviceTransactionConfig({
     });
   }
 
+  if (opReturnData) {
```

Comment:
Could you please revert the change related to op_return in this PR. As it is not related. Thanks